holidayextras/brands
# Hx font bold WIP

#### What does this PR do? (please provide any background)
This applies to both `brand-hx` and `brand-hunter`.
* Import `HolidayExtrasSans-Medium`
* Assign it to `font-weight: bold;`
* Change the `@font-face` name from 'HolidayExtrasSans-Light' to 'HolidayExtrasSans'.
* Bump the versions

#### How can this be tested?
`cd pattern-library && npm link ../brands`
Check out any page that has bold text - e.g. http://localhost:8687/playground/landing-page/index.html
Switch to both HX and Hunter brands.


#### Any tech debt?
Nope

#### Screenshots / Screencast
<img width="780" alt="screen shot 2017-03-10 at 18 23 04" src="https://cloud.githubusercontent.com/assets/400087/23807735/af87dec4-05be-11e7-9eab-36408ed2e4ca.png">


#### What gif best describes how you feel about this work?
![](http://i.giphy.com/3o7TKGn7rtY4ymMDQI.gif)


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
